client/gutenberg/extensions/related-posts/style.scss

```diff
@@ -0,0 +1,34 @@
+.wp-block-a8c-related-posts {
+	[data-align='full'] & {
```

Comment:
In the `save` we should probably rely on classes instead but this is good for `edit` :thumbsup:


Example:
https://github.com/WordPress/gutenberg/blob/5f9107e3fa8fcc5fb23bc169ac972114c4b62381/packages/block-library/src/image/index.js#L206

Reply:
👍 Used a classname in https://github.com/Automattic/wp-calypso/pull/26530/commits/f86daa3a7b3ad99f176c8bab9cc2a08300970edc.

Also used the opportunity there to move the modifier classes a level above, as this way they'll be more useful in case we want to target the top level.